### ISSUE TITLE
Tag statistics

### ISSUE THREAD

**Issue by haesleinhuepf** [NFDI4BIOIMAGE/training#130]
After merging #127 we should look into #101 again. Reason: The tag statistics look different, after data cleaning. 

It might also be cool to have the tag statistics on our website, e.g. as a word cloud. Therefore, we need to execute the code in the notebook as part of a github workflow. Therefore we need to convert the notebook into a python script.

**Comment by SeverusYixin** [comment]
I didn't get it here, after run the script, the yml files should be rewrite, and for the actions show me that this step have been done. 
And I run this tag_statistics.ipynb on my local side, it show me the different result like this :  

Bioimage Analysis: 131
Research Data Management: 76
Python: 64
Artificial Intelligence: 26
Neubias: 26
Nfdi4Bioimage: 23
Omero: 19
Fair-Principles: 16
Imagej: 16
Napari: 12
Workflow Engine: 12
Deep Learning: 9
Sharing: 7
Meta Data: 6
Bio-Image Analysis: 5
Licensing: 5
Microscopy Image Analysis: 5
Cellprofiler: 4
Bioinformatics: 3
Fiji: 3
Segmentation: 3
Teaching: 3
Citing: 2
Conda: 2
Data Protection: 2
...
Science Communication: 1
Workflow: 1
Zarr: 1
Zenodo: 1
Output is truncated. View as a [scrollable element](command:cellOutput.enableScrolling?76c0b664-89f7-4973-8bd6-ae5759bd5edf) or open in a [text editor](command:workbench.action.openLargeOutput?76c0b664-89f7-4973-8bd6-ae5759bd5edf). Adjust cell output [settings](command:workbench.action.openSettings?%5B%22%40tag%3AnotebookOutputLayout%22%5D)...

**Comment by SeverusYixin** [comment]
@haesleinhuepf  Seems this is also finished.

**Comment by SeverusYixin** [comment]
It looks like you want the introduction to the tag statistics to look like a word cloud. Am I right?
![1](https://github.com/user-attachments/assets/17b9eba6-b5f4-4664-956e-a3f371de99bc)

**Comment by haesleinhuepf** [comment]
Yes! git-bob please think about how to write python code for visualizing a word-cloud from a dictionary of python string:int pairs. The number represents how often the string appeared in our dataset and the word-cloud should show popular words accordingly.

**Comment by haesleinhuepf** [comment]
git-bob comment

**Comment by SeverusYixin** [comment]
Also add it to the old Version
